packages/ui/src/composables/useForm.ts

```diff
@@ -12,48 +14,15 @@ export const useFormPropsWithId = {
 }
 
 /**
- * Create `readonly` and `disabled` BEM modifiers.
- * @param props component props
- * @param prefix string with which classes starts (and ends with form state BEM modifier)
+ * @description creates `readonly` and `disabled` BEM modifiers.
+ * @param prefix string that classes start with (base BEM class).
+ * @param props component props.
+ * @returns computed classes object starting with `prefix` and ending with form state BEM modifier.
  */
-export const useForm = <Prefix extends string>(
+export const useForm = (
+  prefix = '',
   props: ExtractPropTypes<typeof useFormProps>,
```

Comment:
Optional parametr must be after non-optional

```suggestion
export const useForm = (
  props: ExtractPropTypes<typeof useFormProps>,
  prefix = 'va',
```

Reply:
It's not optional. `prefix = ''` is here in purpose not to write ${prefix || ''} below.